dockerfile-generator.rb

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env ruby
```

Comment:
I guess, what's the point? 
Ya either add stuff to `Dockerfile` or `Dockerfile.yml`

The formatting can be handled manually, imo. I don't think someone should _have_ to have ruby installed to build any docker image. (docker is all about trying to have as few dependencies as possible)